.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -126,11 +126,15 @@ cross_TextractExplorer:
     - sqs
     - textract
 cross_SubmitDataApp:
-  title: Build an app to submit data to an &DDB; table
-  title_abbrev: Build an app to submit data to an &DDB; table
+  title: Build an application to submit data to a &DDB; table
```

Comment:
The descriptions here don't seem to match the readme.md application description.

Reply:
Not related to Readme